milner_desvars/gazebo.md

```diff
@@ -0,0 +1,71 @@
+#Notes
+
+##Client and server separation
+
+When you run Gazebo you are actually running to different executables. In one hand "gzserver" and in the other "gzclient".
+
+The gzserver executable runs the physics update-loop and sensor data generation. This is the core of Gazebo and it can be used independtly of a graphical interphase. 
+
+The gzclient executable runs a QT based user interface. This application provides a nice visualization of simulation, and convenient controls over various simulation properties.
+
+## World files
```

Comment:
Could you check at least one example of these in the repo?

Reply:
For example, an example of a world file, model file, and plugin. Don't worry if you don't find a plugin.